modules/google-analytics/vue/plugins/googleAnalytics.server.ts

```diff
@@ -1,15 +1,22 @@
 import type { App } from 'vue'
 import GOOGLE_ANALYTICS_ID from '#ioc/config/googleAnalytics/GOOGLE_ANALYTICS_ID'
 
+const usePartytownConfig = import.meta.glob('#ioc/config/PARTYTOWN_CONFIG', { eager: true })
```

Comment:
Nope. google-analytics module cant depend on partytown module. Look how google-tag-manager module does it.